flake.nix

```diff
@@ -38,7 +38,7 @@
     };
 
     barretenberg = {
-      url = "github:AztecProtocol/barretenberg";
+      url = "github:AztecProtocol/barretenberg/76482829f72f4cd9c866737c7501f5c58d86bc6e";
```

Comment:
Can you use the override-input way of changing this?

The commit from Noir master is now: 65e651d04c6092cb5ca079cd9e12ed9b5846fa3a